CSVSource/src/test/java/io/vantiq/extsrc/CSVSource/TestCSV.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Vantiq, Inc.
+ *
+ * All rights reserved.
+ * 
+ * SPDX: MIT
+ */
+
+package io.vantiq.extsrc.CSVSource;
+
+import static org.junit.Assume.assumeTrue;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqResponse;
+import io.vantiq.extsrc.CSVSource.exception.VantiqCSVException;
+
+public class TestCSV extends TestCSVBase {
+    
+    static final int CORE_START_TIMEOUT = 10;
+    static CSVCore core;
+    static CSV csv;
+    static Vantiq vantiq;
+    static Map<String,Object> config ;
+    static Map<String,Object> options ;
+
+    
+    @Before
+    public void setup() {
+        config = new HashMap<String,Object>(); 
+        
+        TestCSVConfig o = new TestCSVConfig();
+        config = o.minimalConfig();
+        options = o.createMinimalOptions();
+
+        csv = new CSV();
+
+        vantiq = new io.vantiq.client.Vantiq(testVantiqServer);
+        vantiq.setAccessToken(testAuthToken);
+    }
+    
+    
+    
+    @Test
+    public void testCorrectErrors() throws VantiqCSVException {
+        
```

Comment:
lots of empty space.  I won't comment on each extra line -- I expect a few to show that this code, for example, should look like

```
    public void testCorrectErrors() throws VantiqCSVException {
         assumeTrue(testFileFolderPath != null && testFullFilePath != null ) ;
         csv.setupCSV(null,testFileFolderPath, testFullFilePath,config,options);
    }

Reply:
done